mne/report.py

```diff
@@ -965,6 +967,96 @@ def add_bem_to_section(self, subject, caption='BEM', section='bem',
         self._sectionlabels.append(sectionvar)
         self.html.extend(html)
 
+    def add_slider_to_section(self, figs, captions=None, section='custom',
+                              scale=None, image_format='png'):
+        """Renders a slider of figs to the report.
+
+        figs : list of figures.
+            Each figure in the list can be an instance of
+            matplotlib.pyplot.Figure, mayavi.core.scene.Scene,
+            or np.ndarray (images read in using scipy.imread).
+        captions : list of str | float | None
+            A list of captions to the figures. If float, a str will be
+            constructed as `Data: %f ms`. If None, it will default to
```

Comment:
thanks. it was ms but I changed it internally, will edit. are these reasonable formats?